Command/ActivateUserCommand.php

```diff
@@ -48,7 +60,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $username = $input->getArgument('username');
 
-        $manipulator = $this->getContainer()->get('fos_user.util.user_manipulator');
+        $manipulator = $this->userManipulator;
```

Comment:
`$this->userManipulator->activate($username);`

Reply:
I'm trying to make the diff as small as possible, to help review this and get a faster merge. I can definitely make these changes if you want.

Reply:
@weaverryan Yes please :)